tests/casp/rcshell_exit_sym.pm

```diff
@@ -0,0 +1 @@
+rcshell_exit.pm
```

Comment:
Yes, feels like a hack to me too. But I have no better way how to run same module multiple times. Maybe I can move code do functions later...